DataConnectors/Digital shadows/digitalshadowsARM.json

```diff
@@ -161,10 +164,10 @@
                         "WorkspaceID": "[parameters('WorkspaceID')]",
                         "WorkspaceKey": "[parameters('WorkspaceKey')]",
                         "digitalshadowsAccountID": "[parameters('digitalshadowsAccountID')]",
-                        "digitalshadowsUrl": "[parameters('digitalshadowsUrl')]",
+                        "historicaldays": "[parameters('historicaldays')]",
```

Comment:
I see the 'digitalshadowsUrl' is removed, why, don'r we want this to be configured by user.

Reply:
It is just a base api url(seen in constant.py as URL) which is same for everyone, it can be kept constant and hard coded since it will not change for different values, like keys and secret. I can add it if it is required.